pkg/xds/bootstrap/generator.go

```diff
@@ -161,6 +162,7 @@ func (b *bootstrapGenerator) Generate(ctx context.Context, request types.Bootstr
 			params.IsGatewayDataplane = true
 		}
 		kumaDpBootstrap.NetworkingConfig.IsUsingTransparentProxy = dataplane.IsUsingTransparentProxy()
+		kumaDpBootstrap.NetworkingConfig.Address = dataplane.Spec.GetNetworking().GetAddress()
```

Comment:
there is something weird about passing this but `IsUsingTransparentProxy` is passed the same way.